Chrono/Chrono/ViewRouter.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ViewRouter.swift
+//  Chrono
+//
+//  Created by Sagar on 2021-06-23.
+//
+
+import Foundation
+import SwiftUI
+
+enum ViewType {
+    case main, onboarding
+}
+
+class ViewRouter: ObservableObject {
+    init() {
+        // Use this only to test!!
+        UserDefaults.standard.set(false, forKey: "didLaunchBefore")
```

Comment:
Should this be commented out? Im guessing right now didLaunchBefore gets set to false whenever the app opens.

Reply:
Fixed